Admin/Get-EASMailboxLogs.ps1

```diff
@@ -6,9 +6,28 @@ param(
     [Parameter(Mandatory = $true)] [string[]] $Mailbox = "",
     [ValidateScript({ Test-Path $_ })]
     [Parameter(Mandatory = $true)] [string] $OutputPath = (Get-Location).Path,
-    [Parameter(Mandatory = $false)] [int] $Interval = 30
+    [Parameter(Mandatory = $false)] [int] $Interval = 30,
+    [Parameter(Mandatory = $false)] [string] $EnableMailboxLoggingVerboseMode = ""
```

Comment:
Seems odd to have a string parameter that will be interpreted as a bool. Why not just make the parameter bool or switch? I think switch would be more natural here.

Edit: See https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_advanced_parameters

![image](https://user-images.githubusercontent.com/4518572/163604533-b2ba713c-e44d-457b-9693-63a59c75738e.png)

Reply:
Bill, point is: when you use Boolean value it is always true or false (be default).
It means that we need somehow understand if False was set by Admin or by Power Shell. As a result, we can override web.config True setting, set in other debugging session.
In other words, I prefer not to touch web.config until I see that False was obviously set by Admin.

Reply:
I see. So either $true or $false could lead to a config change, so we need a third value. Hmm, can we have nullable bool in PowerShell, like `bool?` in C#?

Apparently not. This StackOverflow answer suggests using a ValidateSet with 3 values, true, false, or null: https://stackoverflow.com/questions/51207288/is-it-possible-to-have-nullablebool-in-a-bool-in-powershell

Another option would be two separate switches: -EnableMailboxLoggingVerboseMode and -DisableMailboxLoggingVerboseMode, and make them mutually exclusive.